applications/IgaApplication/custom_conditions/load_condition.cpp

```diff
@@ -0,0 +1,176 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Tobias Teschemacher
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_conditions/load_condition.h"
+
+
+namespace Kratos
+{
+    void LoadCondition::CalculateAll(
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector,
+        ProcessInfo& rCurrentProcessInfo,
+        const bool CalculateStiffnessMatrixFlag,
+        const bool CalculateResidualVectorFlag
+    )
+    {
+        const auto& r_geometry = GetGeometry();
+        const SizeType number_of_nodes = r_geometry.size();
+        const SizeType mat_size = r_geometry.WorkingSpaceDimension() * number_of_nodes;
+
+        // Memory allocation
+        if (CalculateStiffnessMatrixFlag) {
+            if (rLeftHandSideMatrix.size1() != mat_size) {
+                rLeftHandSideMatrix.resize(mat_size, mat_size, false);
+            }
+            noalias(rLeftHandSideMatrix) = ZeroMatrix(mat_size, mat_size);
+        }
+        if (CalculateResidualVectorFlag) {
+            if (rRightHandSideVector.size() != mat_size) {
+                rRightHandSideVector.resize(mat_size, false);
+            }
+            rRightHandSideVector = ZeroVector(mat_size);
+        }
+
+        // Calculation of Force vector
+        if (CalculateResidualVectorFlag) {
+            Vector f = ZeroVector(mat_size);
+
+            // Integration
+            const GeometryType::IntegrationPointsArrayType& integration_points = r_geometry.IntegrationPoints();
+
+            Vector determinat_jacobian_vector(integration_points.size());
+            r_geometry.DeterminantOfJacobian(determinat_jacobian_vector);
+
+            // Shape function values for all integration points
+            const Matrix& r_N = r_geometry.ShapeFunctionsValues();
+
+            for (IndexType point_number = 0; point_number < integration_points.size(); point_number++)
+            {
+                // Differential area
+                const double integration_weight = integration_points[point_number].Weight();
+
+                const double d_weight = integration_weight * determinat_jacobian_vector[point_number];
+
+                // Split only due to different existing variable names
+                // No check included, which checks correctness of variable
+
+                // Point loads
+                if (this->Has(POINT_LOAD))
+                {
+                    const array_1d<double, 3> point_load = this->GetValue(POINT_LOAD);
```

Comment:
```suggestion
                    const array_1d<double, 3>& point_load = this->GetValue(POINT_LOAD);
```
And therefore should be a r_point_load